automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -115,14 +114,15 @@ final class LeonardoSuite
 
 final class LeonardoTerraDockerSuite
     extends Suites(
-      new NotebookAouSpec,
```

Comment:
Mentioned in slack.. do we know why these are failing? Is the test logic out of date with the current images or do we still have some dependency/selenium issues?